packages/libpcap/build.sh

```diff
@@ -3,8 +3,9 @@ TERMUX_PKG_DESCRIPTION="Library for network traffic capture"
 TERMUX_PKG_LICENSE="BSD"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=1.10.0
-TERMUX_PKG_SRCURL=https://fossies.org/linux/misc/libpcap-${TERMUX_PKG_VERSION}.tar.xz
```

Comment:
The .tar.xz works fine when I try it now.

.tar.xz is smaller than .tar.gz and hence preferable

Reply:
Yes it's working now, maybe just because of my vpn problem, will revert this.